prac_07/languages.csv

```diff
@@ -0,0 +1,6 @@
+Language,Typing,Reflection,Year
+Java,Static,Yes,1995
+C++,Static,No,1983
+Python,Dynamic,Yes,1991
+Visual Basic,Static,No,1991
+Ruby,Dynamic,Yes,1995
```

Comment:
Oh, I see, You change the language with class but have not even put in an extra programming language to file, as well as the Pointer Arithmetic.